components/stacks-rpc-client/src/rpc_client.rs

```diff
@@ -58,12 +61,14 @@ pub struct PoxInfo {
     pub contract_id: String,
     pub pox_activation_threshold_ustx: u64,
     pub first_burnchain_block_height: u64,
+    pub current_burnchain_block_height: u64,
     pub prepare_phase_block_length: u32,
     pub reward_phase_block_length: u32,
     pub reward_slots: u32,
     pub reward_cycle_id: u32,
     pub total_liquid_supply_ustx: u64,
-    pub next_cycle: PoxCycle,
+    pub current_cycle: CurrentPoxCycle,
+    pub next_cycle: NextPoxCycle,
```

Comment:
I missed the whole conversation yesterday, but didn't Ludo say these may need to be Optional? Feel free to resolve this comment without addressing it if you're confident that this is okay.